app/dashboard/views.py

```diff
@@ -3517,13 +3517,6 @@ def connect_facebook():
 @login_required
 @require_POST
 def request_verify_facebook(request, handle):
-    is_logged_in_user = request.user.is_authenticated and request.user.username.lower() == handle.lower()
```

Comment:
shouldnt a user be logged in to do this?  were u just testing here?

Reply:
ah my mistake! I thought it all will check in `@login_required` and those are extra.
But `@login_required` won't check if the user is verifying facebook for it's own profile or not.

I will get it back.